### PR TITLE
Add merchantData to response signature priorities

### PR DESCRIPTION
**Pull request by jiripudil**

### CODE REVIEW

File: src/Message/Signature.php [hunk 1]
```diff
@@ -67,7 +67,7 @@ class Signature
 		'resultMessage',
 		'paymentStatus',
 		'authCode',
-		'cardToken',
+		'merchantData',
```
Comment (fprochazka) [inline src/Message/Signature.php:70]: ron proud

**Comment by fprochazka** [comment]
cc @soukiii :)

**Comment by soukicz** [comment]
kitgraphic1

**Pull request merged**